src/OneExplorer.ts

```diff
@@ -65,30 +65,41 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     }
   }
 
-  // TODO(dayo): enable refresh command
   refresh(): void {
     this._onDidChangeTreeData.fire();
   }
 
   getTreeItem(element: OneNode): vscode.TreeItem {
-    element
-        .command = {command: 'oneExplorer.openFile', title: 'Open File', arguments: [element.node]};
     return element;
   }
 
   getChildren(element?: OneNode): OneNode[]|Thenable<OneNode[]> {
-    if (this.oneTree === undefined) {
-      vscode.window.showInformationMessage('No ONE model or config in empty workspace');
-      return Promise.resolve([]);
+    if (!element) {
+      if (!this.workspaceRoot) {
+        vscode.window.showInformationMessage(`Cannot find workspace root.`);
+        return Promise.resolve([]);
+      } else {
+        return Promise.resolve(this.getNode(this.getTree(this.workspaceRoot)));
+      }
     }
 
-    if (element) {
+    if (this.pathExists(element.node.uri.fsPath)) {
       return Promise.resolve(this.getNode(element.node));
     } else {
-      return Promise.resolve(this.getNode(this.oneTree));
```

Comment:
You can update `this.oneTree` or if you don't use `this.oneTree` please remove this variable.

Reply:
Sure. PTAL, #534